cpp/src/arrow/compute/api_scalar.h

```diff
@@ -407,6 +411,18 @@ struct ARROW_EXPORT WeekOptions : public FunctionOptions {
   bool first_week_is_fully_in_year;
 };
 
+struct ARROW_EXPORT Utf8NormalizeOptions : public FunctionOptions {
```

Comment:
We should add this to function_test.cc: https://github.com/apache/arrow/blob/736a69ee7c0b0b059e08b99783a81067beca99fa/cpp/src/arrow/compute/function_test.cc#L40